### DISCUSSION TITLE
Change cover

### DISCUSSION THREAD

**Discussion by mariakherlakian** [joaomlourenco/novathesis#505] [Q&A]
Hello everyone, I am part of a completely different University, but I like this template. Is there any easy way that I can update to my own cover? I tried a few things and it didn't work.

**Comment by joaomlourenco** [comment]
Hi and welcome to the NOVAthesis community. 

Please have a look at file `Config\4_files.fex`, in the very bottom you can fine these lines commented.  Uncomment the ones with `\ntaddfile ` and customize. 

```latex
%%------------------------------------------------------------
% User defined cover instead of standard one
%   syntax: \ntaddfile{cover}[1,N,spine]{filename}
% \ntaddfile{cover}[1]{cover-front-file}  % front cover
% \ntaddfile{cover}[N]{cover-back-file}   % back cover
% \ntaddfile{cover}[spine]{spine-file} % book spine
```
